src/sas/qtgui/Utilities/GuiUtils.py

```diff
@@ -530,18 +539,8 @@ def openLink(url):
         raise AttributeError(msg)
 
 def showHelp(url):
-    """
-    Open a local url in the default browser
-    """
-    location = HELP_DIRECTORY_LOCATION + url
-    #WP: Added to handle OSX bundle docs
-    if os.path.isdir(location) == False:
-        sas_path = os.path.abspath(os.path.dirname(sys.argv[0]))
-        location = sas_path+"/"+location
-    try:
-        webbrowser.open('file://' + os.path.realpath(location))
-    except webbrowser.Error as ex:
-        logging.warning("Cannot display help. %s" % ex)
+    pass
+# showHelp() IS TO BE DELETED IN NEXT COMMIT
```

Comment:
Removing an existing method can be dangerous in case it is used in places you haven't found. Adding a debugging message showing the method is deprecated and then a call to the new method would be appropriate for now.